app.py

```diff
@@ -1,8 +1,14 @@
+import constants
+import window as w
 import pygame
 import random
 import time
 import math
 import pygame.font
+import enemy
+import enemy_tools
+import spaceship
+import utils
```

Comment:
As you know, you can "import" things in Python.

All the other files I've created can be imported by just using their filename.

For example, if I create banana.py with some function "eat_banana() I can do:

```python
# In a different file 
import banana

banana.eat_banana()
```
I split up some of the stuff from the big file into separate smaller files so that you don't have to scroll around as much, and so you can begin using a split monitor set up (if you want) to have code both on the left and right halves of your screen with two files open at once.